src/coreclr/jit/valuenumfuncs.h

```diff
@@ -13,9 +13,9 @@ ValueNumFuncDef(MapSelect, 2, false, false, false)  // Args: 0: map, 1: key.
 ValueNumFuncDef(FieldSeq, 2, false, false, false)   // Sequence (VN of null == empty) of (VN's of) field handles.
 ValueNumFuncDef(NotAField, 0, false, false, false)  // Value number function for FieldSeqStore::NotAField.
 
-ValueNumFuncDef(PtrToLoc, 2, false, false, false)           // Pointer (byref) to a local variable.  Args: VN's of: 0: var num, 1: FieldSeq.
+ValueNumFuncDef(PtrToLoc, 2, false, true, false)            // Pointer (byref) to a local variable.  Args: VN's of: 0: var num, 1: FieldSeq.
```

Comment:
It is for consistency. There are no cases today where it would matter.

Theoretically it could matter in a case like the following:
```cs
ref var locPtr = ref local;

Call(ref locPtr.Field); // Morph will insert a NULLCHECK here
```
We could delete the `NULLCHECK` if we gave `locPtr` a `PtrToLoc` VN with this change, but we don't (for reasons that are mostly historical and have to do with how it is actually used in VN).

It is a correct change though. `PtrToLoc == 0` will always be `false`.